Gruntfile.js

```diff
@@ -60,6 +63,17 @@ module.exports = function (grunt) {
                 'build/js/build.js',
             ],
         },
+        cssmin: {
+              target: {
+                files: [{
+                  expand: true,
+                  cwd: 'build/css',
+                  src: ['*.css'],
```

Comment:
This will pick up all the .css files from the root folder. It would make sense for it to be something like src/css/*.css